### PR TITLE
Add climate meter and entities

### PR DESCRIPTION
**Pull request by amitaibu**
#732

### CODE REVIEW

File: negawatt/modules/custom/climate_meter/climate_meter.features.field_base.inc [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ */
+
+/**
+ * Implements hook_field_default_field_bases().
+ */
+function climate_meter_field_default_field_bases() {
+  $field_bases = array();
+
+  // Exported field_base: 'field_has_data'
+  $field_bases['field_has_data'] = array(
```
Comment (amitaibu) [inline negawatt/modules/custom/climate_meter/climate_meter.features.field_base.inc]: `field_meter_has_data`

File: negawatt/modules/custom/negawatt_climate/negawatt_climate.install [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    'description' => 'The base table for climate meter raw data.',
+    'fields' => array(
+      'id' => array(
+        'description' => 'The primary identifier for the row.',
+        'type' => 'serial',
+        'unsigned' => TRUE,
+        'not null' => TRUE,
+      ),
+      'timestamp' => array(
+        'description' => 'timestamp of recording (for a meter, e.g. modbus) or beginning of period (for period consumption, e.g. iec meter).',
```
Comment (amitaibu) [inline negawatt/modules/custom/negawatt_climate/negawatt_climate.install]: fix

File: negawatt/modules/custom/negawatt_restful/plugins/restful/climate/1.0/NegawattClimateResource.class.php [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    );
+
+    $public_fields['datatime'] = array(
+      'property' => 'datetime'
+    );
+
+    $public_fields['frequency'] = array(
+      'property' => 'frequency',
+      // To prevent conflict with og_membership 'frequency' field.
+      'column_for_query' => 'negawatt_climate_normalized.frequency',
```
Comment (amitaibu) [inline negawatt/modules/custom/negawatt_restful/plugins/restful/climate/1.0/NegawattClimateResource.class.php]: needed?

File: negawatt/modules/custom/occupancy_meter/occupancy_meter.info [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+features[node][] = occupancy_meter
+features[variable][] = comment_anonymous_occupancy_meter
+features[variable][] = comment_default_mode_occupancy_meter
+features[variable][] = comment_default_per_page_occupancy_meter
+features[variable][] = comment_form_location_occupancy_meter
+features[variable][] = comment_occupancy_meter
+features[variable][] = comment_preview_occupancy_meter
+features[variable][] = comment_subject_field_occupancy_meter
+features[variable][] = field_bundle_settings_node__occupancy_meter
+features[variable][] = menu_options_occupancy_meter
```
Comment (amitaibu) [inline negawatt/modules/custom/occupancy_meter/occupancy_meter.info]: remove

**Comment by shahardo** [comment]
Merged to master-0.7

**Pull request closed without merging**